src/engraving/infrastructure/mscreader.cpp

```diff
@@ -407,7 +407,7 @@ ByteArray MscReader::DirReader::fileData(const String& fileName) const
     muse::io::path_t filePath = m_rootPath + "/" + fileName;
     File file(filePath);
     if (!file.open(IODevice::ReadOnly)) {
-        LOGE() << "failed open file: " << filePath;
+        LOGD() << "failed open file: " << filePath;

```

Comment:
Wait, this change is questionable. This message _may_ be relevant when debugging a problem for a user, and LOGD messages are not included in log files for release builds by default. So I suggest using `LOGE()` here.
(We could add a parameter to this method to tell whether the presence of this file is crucial, and based on that choose between LOGD and LOGE, but I don't think that's worth it.)

Reply:
done